packages/block-editor/src/components/block-alignment-matrix-toolbar/index.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { DOWN } from '@wordpress/keycodes';
+import {
+	Button,
+	Dropdown,
+	ToolbarGroup,
+	__experimentalAlignmentMatrixControl as AlignmentMatrixControl,
+} from '@wordpress/components';
+
+export function BlockAlignmentMatrixToolbar( props ) {
+	const {
+		label = __( 'Change matrix alignment' ),
+		onChange = noop,
+		value = 'center',
+	} = props;
+
+	const icon = <AlignmentMatrixControl.Icon value={ value } />;
+	const className = 'block-editor-block-alignment-matrix-toolbar';
+
+	return (
+		<Dropdown
+			position="bottom right"
+			className={ className }
+			popoverProps={ { className } }
```

Comment:
I'm pretty sure you shouldn't be using the same `className` for both of these.

Reply:
It is needed to target the Dropdown (popover) menu, since the menu is being Portal'ed elsewhere in the DOM.

Reply:
Maybe I'm misunderstanding what you mean, but it should still use a distinct `className`, shouldn't it?

Reply:
I suppose it could. I'll make that adjustment